messages/flow.run.test.md

```diff
@@ -1,51 +1,49 @@
 # summary
 
-Summary of a command.
+Invoke flow tests in an org.
 
 # description
 
-More information about a command. Don't repeat the summary.
+Specify which tests to run by using the --class-names flag followed by the names of the flows you want to test. For example, if you save a flow with the name Flow1, then use: --class-names Flow1.
 
-# examples
-
-- <%= config.bin %> <%= command.id %>
-
-# flags.result-format.summary
+To see code coverage results, use the --code-coverage flag with --result-format. The output displays a high-level summary of the test run and the code coverage values for classes in your org. If you specify human-readable result format, use the --detailed-coverage flag to see detailed coverage results for each test method run.
 
-Format of the test results.
+By default, "flow run test" runs asynchronously and immediately returns a test run ID. If you use the -–synchronous flag, you can use the --wait flag to specify the number of minutes to wait; if the tests finish in that timeframe, the command displays the results. If the tests haven't finished by the end of the wait time, the command displays a test run ID. Use the "flow get test --test-run-id" command to get the results.
```

Comment:
```suggestion
By default, "flow run test" runs asynchronously and immediately returns a test run ID. If you use the -–synchronous flag, you can use the --wait flag to specify the number of minutes to wait; if the tests finish in that timeframe, the command displays the results. If the tests haven't finished by the end of the wait time, the command displays a test run ID. Use the "flow get test --test-run-id" command to get the results.

```

Reply:
I added a new-line between the two paragraphs so the `--help` is easier to read.